_posts/2015-08-11-one-month-as-an-epagee.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "One month as an ePagee"
+date: "2015-08-11 08:00:00"
+icon: user-plus
+tags: induction employee welcome introduction onboarding
+categories: jobs
+authors: ["Mathias", "Mats"]
+---
+
+Starting a new job is always an adventure. You made a careful decision for a new job and now the first day is approaching. This day is especially exciting because it will give you the first impression if the decision for this particular position was the right one.
+
+So it was really a little relief to receive a very warm welcome on our first day. Dirk and Harm said hello from the management side and introduced ePages. Leana from HR made sure that all the organisational stuff was taken care of and Keun-Man provided us with the hardware. So we were basically ready to start. Afterwards our buddies introduced us to everybody - ePages is a very diverse company. The team is very international and has a good mix of age groups. You find people from Spain, Italy, Belarus, Brasil, people that are on their first job and also more experienced ones - and even parents ;-).
```

Comment:
"on our first day" <-- you didn't mention that there were others who started with you. Suggestion: "So when me and three other new employees had our first day, it was really a little relief to receive a very warm welcome."